modules/custom/express_permissions/express_permissions.module

```diff
@@ -264,6 +264,9 @@ function express_permissions_secure_permissions($role) {
       'view scheduled content',
       'administer matchredirect',
       'use add another',
+      // add Masquerade Permissions
+      'masquerade as user',
+      'masquerade as any user',
```

Comment:
Can a user masquerade as user 1?

Reply:
No, a user cannot masquerade as user 1.
Administrators: If you are an admin and go to user/1 page, you get Access Denied. 
Developer: If you are a developer and go to /user/1 page, then yes, you see the 'masquerade as user 1' buttons. But when you click on it in order to begin the masquerade, you get Access Denied. 
No other role has masquerade ability.